qiskit/circuit/quantumcircuit.py

```diff
@@ -2273,6 +2279,29 @@ def cz(self, control_qubit, target_qubit, *, label=None, ctrl_state=None,
         return self.append(CZGate(label=label, ctrl_state=ctrl_state),
                            [control_qubit, target_qubit], [])
 
+    def add_calibration(self, gate, qubits, schedule, params=None):
+        """Register a low-level, custom pulse definition for the given gate.
+
+        The custom pulse definition of a given gate is of the form
+            {'gate_name': {(qubits, params): schedule}}
```

Comment:
Move this comment to `calibrations` property docstring?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4859/commits/0e719ebb387a8187fa06d39b7c642d5185dc35cc.